_assets/ci/Jenkinsfile

```diff
@@ -1,79 +1,101 @@
+              sh 'make statusgo-android'
+            } } }
+            stage('Archive') { steps {
+              sh """
+                mv ${env.STATUS_PATH}/build/bin/statusgo-android-16.aar \
+                  ${env.WORKSPACE}/status-go-android-${lib.sufix()}.aar
+              """
+              sh 'ls -l'
+              archiveArtifacts "status-go-android-${lib.sufix()}.aar"
+            } }
+            stage('Upload') { steps { script {
+              lib.uploadArtifact("status-go-android-${lib.sufix()}.aar")
+            } } }
+          }
+        }
+        stage('iOS') {
+          stages {
+            stage('Compile') { steps { dir(env.STATUS_PATH) {
+              sh 'make statusgo-ios-simulator'
+              dir('build/bin/statusgo-ios-9.3-framework') {
+                sh 'zip -r status-go-ios.zip Statusgo.framework'
+              }
+            } } }
+            stage('Archive') { steps {
+              sh """
+                mv ${env.STATUS_PATH}/build/bin/statusgo-ios-9.3-framework/status-go-ios.zip \
+                  ${env.WORKSPACE}/status-go-ios-${lib.sufix()}.zip
+              """
+              sh 'ls -l'
+              archiveArtifacts("status-go-ios-${lib.sufix()}.zip")
+            } }
+            stage('Upload') { steps { script {
+              lib.uploadArtifact( "status-go-ios-${lib.sufix()}.zip")
+            } } }
+          }
+        }
+      }
     }
-
-    stage('Deploy') {
-        // For branch builds, replace the old artifact. For develop keep all of them.
-        def version = gitBranch == 'develop' ? getVersion(gitBranch, gitShortSHA, '') : getVersion(gitBranch, gitShortSHA, env.BUILD_ID)
-        def server = Artifactory.server 'artifactory'
-        def uploadSpec = """{
-            "files": [
-                {
-                    "pattern": "build/bin/statusgo-android-16.aar",
-                    "target": "libs-release-local/status-im/status-go/${version}/status-go-${version}.aar"
-                },
-                {
-                    "pattern": "build/bin/statusgo-ios-9.3-framework/status-go-ios.zip",
-                    "target": "libs-release-local/status-im/status-go-ios-simulator/${version}/status-go-ios-simulator-${version}.zip"
-                }
-            ]
-        }"""
-
-        def buildInfo = Artifactory.newBuildInfo()
-        buildInfo.env.capture = false
-        buildInfo.name = 'status-go (' + gitBranch + '-' + gitShortSHA + ')'
-        server.upload(uploadSpec, buildInfo)
-        server.publishBuildInfo(buildInfo)
+    stage('Release') { when { expression { params.GITHUB_RELEASE == true } }
+      steps {
+        sh "make prepare-release"
+        withCredentials([[
+          $class: 'UsernamePasswordMultiBinding',
+          credentialsId: 'status-im-auto',
+          usernameVariable: 'GITHUB_USER',
+          passwordVariable: 'GITHUB_TOKEN'
+        ]]) {
+          sh "yes | make release release_branch=${gitBranch}"
```

Comment:
I don't see any line that moves built files to a directory which is used by `github-release` triggered by this target.

Reply:
Wait, isn't that supposed to be done by `make prepare-release`? That is there.

Reply:
Oh yeah. That's true. Sorry, I didn't notice `sh 'make prepare-release'`.